File: CS-Examples/09_Charts/LineChartDroplines.vb
Imports Spire.Xls

Namespace LineChartDroplines
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub
		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
            Using workbook As New Workbook()
                ' Load the Excel file from disk.
                workbook.LoadFromFile("..\..\..\..\..\..\Data\LineChartDroplines.xlsx")

                ' Get the first worksheet in the workbook.
                Dim worksheet As Worksheet = workbook.Worksheets(0)

                ' Get the first chart in the worksheet.
                Dim chart As Chart = worksheet.Charts(0)

                ' Enable drop lines for the first series of the chart.
                chart.Series(0).HasDroplines = True

                ' Save the modified document to "result.xlsx" using the Excel 2013 format.
                workbook.SaveToFile("result.xlsx", FileFormat.Version2013)
            End Using
            ExcelDocViewer("result.xlsx")
		End Sub

		Private Sub ExcelDocViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub
		Private Sub btnClose_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnClose.Click
			Close()
		End Sub
	End Class
End Namespace

File: CS-Examples/09_Charts/LineChartDroplines.cs
using System;
using System.Windows.Forms;
using Spire.Xls;

namespace LineChartDroplines
{
	public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }
		private void btnRun_Click(object sender, System.EventArgs e)
		{
            // Create a new workbook
            Workbook workbook = new Workbook();
            
            // Load Excel from disk
            workbook.LoadFromFile(@"..\..\..\..\..\..\Data\LineChartDroplines.xlsx");

            // Get the first sheet
            Worksheet worksheet = workbook.Worksheets[0];

            // Get the first chart
            Chart chart = worksheet.Charts[0];

            // Add a drop lines to the first series
            chart.Series[0].HasDroplines = true;

            // Save the document
            workbook.SaveToFile("result.xlsx", FileFormat.Version2013);

            // Dispose of the workbook object to release resources
            workbook.Dispose();

            // Launch the file
            ExcelDocViewer("result.xlsx");
		}

        private void ExcelDocViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }
        private void btnClose_Click(object sender, EventArgs e)
        {
            Close();
        }
	}
}
